<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab { width: 243px;}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="productTab">
    <attribute name="title">
     <string>Ürün Listele</string>
    </attribute>
    <widget class="QTreeWidget" name="productsTree">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>302</width>
       <height>321</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Barkod</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ürün İsmi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stok Sayısı</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="getProductsButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ürünleri Getir</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createUserTab">
    <attribute name="title">
     <string>Kullanıcı Oluştur</string>
    </attribute>
    <widget class="QLabel" name="usernameLbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kullanıcı Adı</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="usernameTxt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parola</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="passwordTxt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="accessPermissionLbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Yetki</string>
     </property>
    </widget>
    <widget class="QComboBox" name="accessPermissionComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Kullanıcı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Admin</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="addUserButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ekle</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="logTab">
    <attribute name="title">
     <string>Loglar</string>
    </attribute>
    <widget class="QTextEdit" name="logTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>701</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="updateLogButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Güncelle</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
